rclpy/rclpy/logging.py

```diff
@@ -70,3 +71,8 @@ def get_logger_effective_level(name):
 def get_logging_severity_from_string(log_severity):
     return LoggingSeverity(
         _rclpy_logging.rclpy_logging_severity_level_from_string(log_severity))
+
+
+def get_logging_directory() -> Path:
+    """Return the current logging directory being used."""
```

Comment:
I guess cross-references aren't possible at the moment, but it's probably worth at least copying part of the docs from the `rcl_logging` function related to how the environment variables are used.

Reply:
I will refer to `rcl_logging_get_logging_dir()`.
Though cross references not work and people would need to do an extra search that's better that than out of date docs.